piparty.py

```diff
@@ -654,7 +691,7 @@ def game_loop(self):
                 self.check_for_new_moves()
                 if len(self.tracked_moves) > 0:
                     self.check_new_admin()
-                    self.check_change_mode()
+                    self.check_change_mode() # TODO - do we want to make this so only admins can change mode?
```

Comment:
it's not a bad idea (especially for larger groups, hitting the buttons can disrupt things) but for a smaller group, let's say 4 players, it's easier than going into admin mode every time. This probably should be a setting that can lock switching modes to admins only, so for conventions it would make sense.